tests/mocks/wc-memberships.php

```diff
@@ -0,0 +1,45 @@
+<?php // phpcs:ignoreFile
```

Comment:
Added a mock for wc-memberships plugin testing. Not sure if this is the best way to approach this though, so let me know if there is a cleaner approach!